SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Supported Versions
+
+Only certain branches are supported with security updates.
+
+| Version (branch) | Supported |
+| ---------------- | --------- |
+| main    | :white_check_mark: |
+| other   | :x:                |
+
+When using this code or reporting vulnerability please be sure to use supported branches and the most recent release tag.
+
+## Reporting a Vulnerability
+
+Use the `Report a vulnerability` link at https://github.com/GSA-TTS/docker-trestle/security to report a security vulnerability
```

Comment:
Shouldn't we refer to the GSA program?

Reply:
That's a good question, would the GSA program be a good place to point people for just code libraries (this is a running app, but only locally on a dev's machine, not a web service)? I had the same question/issue with the SECURITY.md file for terraform-cloudgov. The GSA VDP seems to be explicitly limited to running systems that are registered on HackerOne.